addons/mail/static/src/core/common/messaging_service.js

```diff
@@ -418,7 +418,7 @@ export class Messaging {
                 case "ir.attachment/delete":
                     {
                         const { id: attachmentId, message: messageData } = notif.payload;
-                        if (messageData) {
+                        if (messageData?.id) {
```

Comment:
I don't understand the change, in which case do we send a message payload without providing the message id?